cmd/thermal-recorder/config_test.go

```diff
@@ -28,27 +28,25 @@ import (
 	"github.com/stretchr/testify/require"
 	"gopkg.in/yaml.v2"
 
+	"github.com/TheCacophonyProject/thermal-recorder/location"
 	"github.com/TheCacophonyProject/thermal-recorder/motion"
 	"github.com/TheCacophonyProject/thermal-recorder/recorder"
 	"github.com/TheCacophonyProject/thermal-recorder/throttle"
 	"github.com/TheCacophonyProject/window"
 )
 
-var locationFileName = "test_data/location.yaml"
-
-func init() {
-	locationFileName = filepath.Join(GetBaseDir(), "test_data", "location.yaml")
-}
+const (
+	testDir = "test_data"
+)
 
-func TestAllDefaults(t *testing.T) {
-	conf, err := ParseConfig([]byte(""), []byte(""), locationFileName)
-	require.NoError(t, err)
-	require.NoError(t, conf.Validate())
+var testLocationFileName = filepath.Join(GetBaseDir(), testDir, "location.yaml")
+var testConfigFile = filepath.Join(GetBaseDir(), testDir, "config.yaml")
+var testUploaderFile = filepath.Join(GetBaseDir(), testDir, "uploader-config.yaml")
```

Comment:
maybe have a helper to create paths to test config files rather and copying and pasting the same pattern?